### PR TITLE
Unoppresses Mentors pt.1 & 2

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request
now that it benefits me directly i will code for the oppressed mentorbase
FsFkI7H

#### Changelog

:cl:
bugfix: Mentors are now properly shown as such in OOC instead of as donators
bugfix: Mentors can now set their OOC color preference
/:cl:

### CODE REVIEW

**Comment by Readystorm546** [comment]
STEP 1: SECURE THE KEYS

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -114,14 +114,15 @@ GLOBAL_VAR_INIT(mentor_ooc_colour, YOGS_MENTOR_OOC_COLOUR) // yogs - mentor ooc
 			oocmsg += "OOC:</span> <EM>[keyname]:</EM> <span class='message linkify'>[msg]</span></span></font>" // Footer for an admin or AO's OOC.
 			oocmsg_toadmins = oocmsg
 	else
-		oocmsg = "<span class='ooc'>[is_donator(src) ? "(Donator)" : ""]"
 		if(is_mentor()) // If the speaker is a mentor
+			oocmsg = "<span class='ooc'>["(Mentor)"]"
```
Comment (monster860) [inline code/modules/client/verbs/ooc.dm]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

```suggestion
			oocmsg = "<span class='ooc'>(Mentor)"
```

**Review by monster860 — CHANGES_REQUESTED** [review]

File: code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (AsV9) [reply code/modules/client/verbs/ooc.dm]: Now that is big brain

Comment (Hopekz) [reply code/modules/client/verbs/ooc.dm]: what the fuck

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
Donator should probably override mentor.

File: code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (alexkar598) [reply code/modules/client/verbs/ooc.dm]: How the fuck does this compile

**Comment by gilvian** [comment]
Dear nich.
My donator ran out long time ago, mentors even without donator are just pink donators. 
Stop being mean to mentors. 

With love
Gilvian also known as Gibong

**Comment by MegaEmpirical** [comment]
Fix mentor donator perks: for some reason they only have 3 character slots

**Comment by gilvian** [comment]
> Fix mentor donator perks: for some reason they only have a character slots

We dont have slots working

**Review by alexkar598 — CHANGES_REQUESTED** [review]
For the love of god, fix the whitespace changes, I don't say anything when there is 1-2 but this is too much

**Comment by ynot01** [comment]
> For the love of god, fix the whitespace changes, I don't say anything when there is 1-2 but this is too much

all i did was delete unnecessary spacing im saving bytes here valuable bytes

**Pull request merged**